pulp-glue/pulp_glue/common/openapi.py

```diff
@@ -603,6 +658,7 @@ def render_request(
         security: t.List[t.Dict[str, t.List[str]]] = method_spec.get(
             "security", self.api_spec.get("security", {})
         )
+        auth: t.Optional[t.Union[tuple[str, str], requests.auth.AuthBase]] = None
```

Comment:
Is this needed? Why?

Reply:
Maybe mypy was complaining about it ages ago. Removed.